babel.config.js

```diff
@@ -244,6 +244,14 @@ module.exports = function (api) {
             { name: "IS_STANDALONE", value: env === "standalone" },
             "flag-IS_STANDALONE",
           ],
+          [
+            pluginToggleBooleanFlag,
+            {
+              name: "USE_ESM_OR_STANDALONE",
```

Comment:
We can then use this in other packages, so that we don't have to manually check for both cases.